docs/TROUBLE.md

```diff
@@ -0,0 +1,162 @@
+
+- If the LUA script does not work, ensure you have installed the correct version.  There is a separate version for the lower resolution black and white screens.
+
+- Inspect the Tx module connector and ensure your Tx module is fully seated in the bay.
+
+- If your Tx module is the older version of the [R9M](FRSKY_R9.md#r9m-versions), don't forget to do the Inverter Mod.
+
+## 3. Confirm radio link between Tx module and Rx is connected
+
+If the pre-compiled firmware or firmware built with the same bind phrase is installed on new hardware or after erasing the flash for both the Tx module and the Rx, then no bind step is required.
+
+Power up a single Tx module and compatible Rx.
+
+When the Tx module and Rx share the same bind phrase, a LED (usually green) should flash more slowly at 1 Hz (once per second, on for 1/2 second then off for 1/2 second repeatedly) to indicate they are connected and exchanging radio messages.
+
+You should also use the [mLRS LUA script](LUA.md) to confirm the radio link and firmware versions.  It is HIGHLY recommended to use the same firmware version on both the Rx and Tx. The latest released firmware is also recommended, especially for new installs.
+
+- If the radio link does not connect because the Rx was previously configured or bound with a different bind phrase, follow the [Binding](BINDING.md) procedure.
+
+- If the binding procedure doesn't establish the connection, verify that both the Rx and Tx module are compatible which means they must be on the same band (2.4 GHz, 915/866 MHz or 433 MHz). In the case of 915/866 MHz, also check that the LoRa chipsets are [compatible](SX126x_SX127x_INCOMPATIBILITY.md).  For example, the FrSky R9 series and ELRS 900 MHz receivers are not compatible with the MatekSys 900 MHz hardware.
+
+- Be sure you have flashed the correct [firmware file](https://github.com/olliw42/mLRS#firmware-flashing).  Files which start with tx are only for modules functioning in the Tx role and rx files are only for modules functioning in the rx role.  Also, pay attention to the variants for different use cases and hardware configurations.  For example, there are 3 separate files for different hardware and use case options for the MatekSys mr900-30 when used as a Tx module and a 4th file for when the same hardware is used as a Rx.  A module with rx firmware flashed won't connect with any other module with rx firmware flashed and tx firmware won't connect with tx firmware.
+
+## 4. Confirm MAVLink message flow from flight controller to Rx and Tx module
+
+In EdgeTX/OpenTX go to MDL->TELEMETRY and select "Discover new sensors".  Then, scroll up using the scroll wheel and you should find a sensor named FM among several other sensors.  It should have a star flashing next to it indicating it is updating about once per second.  See the [CRSF](CRSF.md] documentation page for background info.
```

Comment:
" should have a star or point flashing ..."
link not working -> [CRSF](CRSF.md)